subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationFilter.kt

```diff
@@ -24,6 +24,8 @@ abstract class InstrumentationFilter(val name: String) {
         private val annotations = Filter<String>()
         private val prefixes = Filter<String>()
 
+        var excludeFlaky = false
```

Comment:
I had to separate flaky filtration from other annotations because AnnotationFilter is able to filter only by annotation name but in Flaky case, we have to parse annotation fields

Reply:
Can we use a more helpful model for annotation in AnnotationFilter?
I suppose that we will have other annotations with parameters.
It would be good to have a common abstraction.

Reply:
I don't see scenarios. 
Users don't need knowledge about Annotations values.
There are two annotation types:
- Marker - you simple use name for filtering
- Complex with logic: Flaky, SkipOnSdk. Filters are written with parsing and custom logic. I consider staying logic separated.
One option I see is the merging of SkipOnSdk and Flaky. We can add a `skip` parameter to Flaky annotation